javascript/builtins/Intl/NumberFormat.json

```
@@ -962,6 +962,192 @@
                 "deprecated": false
               }
             },
+            "result_roundingIncrement_property": {
+              "__compat": {
```

Adding these. Makes it easier to extend later if the compatibility changes.
```suggestion
              "__compat": {
                "description": "The result <code>roundingMode</code> parameter with possible integer values:<code>'1'</code>, <code>'2'</code>, <code>'5'</code>, <code>'10'</code>, <code>'20'</code>, <code>'25'</code>, <code>'50'</code>, <code>'100'</code>, <code>'2000'</code>, <code>'2500'</code>, <code>'5000'</code>.",
```